buf.gen.yaml

```diff
@@ -10,26 +10,25 @@ managed:
       - buf.build/envoyproxy/protoc-gen-validate
       - buf.build/grpc-ecosystem/grpc-gateway
 plugins:
-  - remote: buf.build/protocolbuffers/plugins/go:v1.28.0-1
```

Comment:
Newer buf cli versions use `plugin` key for both remote and local plugin references.

Also, this updates the plugins to their latest versions. Note, these plugins are managed by Buf, and will be replacing the ones referenced here.

New plugins can be found here:

- https://buf.build/protocolbuffers/go
- https://buf.build/grpc/go
- https://buf.build/bufbuild/validate-go
- https://buf.build/grpc-ecosystem/gateway
- https://buf.build/grpc-ecosystem/openapiv2

All of these are managed by Buf, and the build source is available at [bufbuild/plugins](https://github.com/bufbuild/plugins)